README.md

```
@@ -1,3 +1,41 @@
+# AI 기반 일정 및 업무 관리 서비스 (Carrier)
+
+AI가 일정, 메일, 회고, 회의를 간편화해 **업무를 간편하게 만드는 서비스**입니다.  
+**사회초년생의 성장**을 돕고, **통합 플랫폼에서 모든 업무를 해결**하도록 설계되었습니다.
+
+프로젝트 기간 - 2025.01. ~ 2025.04.
+
+[운영중인 서비스 바로가기](https://www.jing5s.kro.kr)
+
+## 기능
+
+**이메일**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)